pycheribuild/projects/cross/freertos.py

```diff
@@ -109,7 +128,7 @@ def setup_config_options(cls, **kwargs):
             help="The FreeRTOS program to build.")  # type: str
 
         cls.demo_bsp = cls.add_config_option(
-            "bsp", metavar="BSP", show_help=True,
+            "demo_bsp", metavar="BSP", show_help=True,
```

Comment:
That's not the naming style. You should perhaps put generic in there.

Reply:
Not sure I get that?

Reply:
None of the other command line flags use an underscore, it should probably be:
```suggestion
            "demo-bsp", metavar="BSP", show_help=True,
```